src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationNewWithForceStrictCondition.java

```diff
@@ -17,11 +17,50 @@
 import java.io.InputStreamReader; //indent:0 exp:0
 
 public class InputIndentationNewWithForceStrictCondition { //indent:0 exp:0
-    void test() throws IOException  { //indent:4 exp:4
+    private java.util.List < ? > [ //indent:4 exp:4
+           ] arrayOfLists; //indent:11 exp:12,33 warn
+    public Object[] //indent:4 exp:4
+            variable1; //indent:12 exp:12
+    int variable2 //indent:4 exp:4
+    []; //indent:4 exp:12 warn
+    int[] //indent:4 exp:4
+    		variable3; //indent:12 exp:12
+
+	void test() throws IOException  { //indent:4 exp:4
         BufferedReader bf =  //indent:8 exp:8
                 new BufferedReader(  //indent:16 exp:16
                 new InputStreamReader(System.in) {  //indent:16 exp:24 warn
                     int a = 0; //indent:20 exp:28,32,36 warn
                 }); //indent:16 exp:24,28,32 warn
+
+        String[] tmp1 = new String[42 //indent:8 exp:8
+                                   + bf.toString().length()]; //indent:35 exp:16,35
```

Comment:
it should be violation 
because `checkConfig.addAttribute("forceStrictCondition", "true");` for this Input

Reply:
In my opinion 35 is also a valid indentation. From https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html in "4.2 Wrapping Lines", fourth point: "Align the new line with the beginning of the expression at the same level on the previous line." In this case the expression begins at position 35 after the `[`.
@romani If you prefer 16 to be the only valid value I can change my code :-)

Reply:
We have Check implementation that can satisfy multiple style guides, you reference sun style (one of many style guides). For this style config should be `checkConfig.addAttribute("forceStrictCondition", "false");`.

But in this particular Test/input we have `checkConfig.addAttribute("forceStrictCondition", "true");` so it mean indentation should be strictly at required position.
You can create another Input+test methods where  `checkConfig.addAttribute("forceStrictCondition", "false");` and this code will be without violation.
https://checkstyle.org/config_misc.html#Indentation

Reply:
I still do not see violation on this line

thee should be "wan" trailing comment and update in UT method in expected array

Reply:
Line 37 is now treated as violation